src/app/tab2/tab2.page.html

```diff
@@ -13,28 +11,22 @@
     </ion-toolbar>
   </ion-header>
 
-  <app-explore-container name="Tab 2 page"></app-explore-container>
-</ion-content>
-
-<ion-col size="6" *ngFor="let photo of photoService.photos; index as position">
```

Comment:
I accidentally duplicated it…? That's why it was showing up double…

Reply:
Yeah is that and the class one element had which basically is a full screen element

Reply:
Class one? I'll have to refresh my memory on that, but basically it's an element outside a code block, right?